app/models/admin_ability.rb

```diff
@@ -120,6 +125,10 @@ def signed_in_with_organizer_role(user, conf_ids_for_organization_admin = [])
     can :manage, Registration, conference_id: conf_ids
     can :manage, RegistrationPeriod, conference_id: conf_ids
     can :manage, Booth, conference_id: conf_ids
+    cannot :accept, Booth do |booth|
```

Comment:
common_abilities_for_roles (https://github.com/nasia-sam/osem/blob/cab940207e346bbf1b957abdfad7217854f25f2c/app/models/admin_ability.rb#L20) doesn't have the can :manage, Booth definition, yet it has the cannot :accept, Booth definition (https://github.com/nasia-sam/osem/blob/cab940207e346bbf1b957abdfad7217854f25f2c/app/models/admin_ability.rb#L78)

More importantly the can :manage, Booth definition depend on other parameters, which are unique to the individual function they appear in. For example, when you are in the organizer role (https://github.com/nasia-sam/osem/blob/cab940207e346bbf1b957abdfad7217854f25f2c/app/models/admin_ability.rb#L109) your definition is:
`can :manage, Booth, conference_id: conf_ids`
https://github.com/nasia-sam/osem/blob/cab940207e346bbf1b957abdfad7217854f25f2c/app/models/admin_ability.rb#L127

But when you are in the cpf role (https://github.com/nasia-sam/osem/blob/cab940207e346bbf1b957abdfad7217854f25f2c/app/models/admin_ability.rb#L178) your definition is:
` can :manage, Booth, conference_id: conf_ids_for_cfp`
https://github.com/nasia-sam/osem/blob/cab940207e346bbf1b957abdfad7217854f25f2c/app/models/admin_ability.rb#L186

So in one case we have the conf_ids_for_cfp and in the other we have conf_ids, each of those two variables hold different values, as such we need to define the can :manage, Booth ability in each function (once for the organizer role, one for the cfp role).

However in all places your definition reads exactly the same, and it does not take into account the role of the user.
```
cannot :accept, Booth do |booth|
      conference = booth.conference
      conference.maximum_accepted_booths?
    end
```

So my question remains:
- Why do we  need to define the exact same block 3 times?
- And based on your previous answer, why is it even defined in common_abilities where there is no can :manage, Booth definition at all?